lte/gateway/c/core/oai/test/mock_tasks/CMakeLists.txt

```diff
@@ -35,4 +33,5 @@ add_library(MOCK_TASKS
 
 target_link_libraries(MOCK_TASKS
     LIB_ITTI gmock_main gtest gtest_main gmock
+    TASK_GRPC_SERVICE
```

Comment:
Though we do not need to link with `TAKS_GRPC_SERVICE` - we do need / want it's PUBLIC headers, and we want to guarantee it is built before this target (to ensure proto / grpc gen has occured).